src/background_thread.c

```diff
@@ -16,6 +16,7 @@ bool opt_background_thread = BACKGROUND_THREAD_DEFAULT;
 malloc_mutex_t background_thread_lock;
 /* Indicates global state.  Atomic because decay reads this w/o locking. */
 atomic_b_t background_thread_enabled_state;
+UNUSED static bool background_thread_enabled_at_fork;
```

Comment:
Maybe all the variables that are only used when ```JEMALLOC_BACKGROUND_THREAD``` is defined should just be in a ```#ifdef``` block.